Python/traceback.c

```diff
@@ -1229,6 +1229,13 @@ dump_traceback(int fd, PyThreadState *tstate, int write_header)
         if (frame == NULL) {
             break;
         }
+        if (frame->owner == FRAME_OWNED_BY_CSTACK) {
+            /* Trampoline frame */
+            frame = frame->previous;
+        }
```

Comment:
It's never possible to have two trampolines in a row, right? I'm thinking of some nightmare scenario where new code starts running during a trampoline frame for some reason (like as part of a `Py_DECREF`, which I don't think can happen).

Reply:
I don't see how this can happen, but I'll add an assert.